### DISCUSSION TITLE
[C# Unity] Algorand SDK for Unity

### DISCUSSION THREAD

**Discussion by jasonboukheir** [WalletConnect/walletconnect-monorepo#3099] [Developer Support]
Hi, I'm working on migrating the Algorand SDK for Unity to version 2.0 of WalletConnect. I have limited resources, so my goals are the following:

- Support minimum scope of WalletConnectV2 to allow for Algorand transaction signing with *most* wallets.
- Support Windows, Mac, Linux, iOS, Android, and WebGL platforms.

Because of the multi-platform nature, I can't use WalletConnectUnity or WalletConnectSharp. In the meantime, I've been asking users to use those projects if they are building to a Mono-friendly platform.

It looks like I'll need to implement Dapp client side protocols of the Sign API, and the Core APIs from the documentation. Am I on the right track? Is there a migration guide somewhere for SDK developers?

**Comment by finessevanes** [comment]
Shared with your Unity team!

**Reply by jasonboukheir** [reply]
Hi @finessevanes , could you clarify where you shared it a little more? Thank you very much for your reply! :)
